.claude/hooks/pre_bash_redirect.py

```diff
@@ -50,8 +50,14 @@
 def main():
     try:
         payload = json.load(sys.stdin)
-    except Exception:
-        sys.exit(0)
+    except Exception as e:
+        # Fail secure: block (exit 2) if we can't parse input to verify safety
+        print(  # noqa: T201
+            f"Policy: Failed to parse hook input ({type(e).__name__}: {e}). "
+            f"Blocking tool call for safety.",
+            file=sys.stderr,
+        )
+        sys.exit(2)
```

Comment:
The exception handler for JSON parsing previously exited with code 0 (allow), which silently bypassed policy enforcement if stdin was malformed or missing. Any parsing error would allow the tool call to proceed unchecked, defeating the security policy intent.

@sfc-gh-nbellante Do you agree with this aspect?